### PR TITLE
Dev 39 - Placeholder course does not save department or tag

### PR DESCRIPTION
**Pull request by Byuljapcho**
**Description**
When a user creates a custom course and adds it to the plan, the department and tag values would not save and show up as "none" when the user opens up the customized course again. 

**Implementation**
In the CourseDisplayPopup.tsx, I changed the department value in the to-be-dispatched new placeholderCourse object from "none" to user-changed department value. I also added the department value to the json body request so that the backend also gets the updated values. In the CourseDisplay.tsx, I added code to update the user-changed tag values in the backend. Finally, in Placeholder.tsx, I added code to set the tag value if it exists and also display the retrieved tag value.  

**Testing**
Tested locally.
Screen Shot 2023-01-27 at 9 48 00 AM

### CODE REVIEW

**Review by mattliu-mygit — CHANGES_REQUESTED** [review]
Please fix prettier issues.

File: lib/components/popups/course-search/search-results/Placeholder.tsx [hunk 1]
```diff
@@ -131,6 +132,7 @@ const Placeholder: FC<{ addCourse: (plan?: Plan) => void }> = (props) => {
     setPlaceholderDepartment(dep);
     if (inspectedVersion !== 'None') {
       const inspCopy: Course = { ...inspectedVersion, department: dep };
+      console.log(inspCopy, 3);
```
Comment (mpark63) [inline lib/components/popups/course-search/search-results/Placeholder.tsx:135]: remove console.log

**Review by mpark63 — CHANGES_REQUESTED** [review]

**Review by mattliu-mygit — APPROVED** [review]
Lgtm!

File: lib/components/popups/course-search/search-results/Placeholder.tsx (hunk 1 above)
Comment (uCredit-Dev) [reply lib/components/popups/course-search/search-results/Placeholder.tsx]: done

**Review by josephined21 — APPROVED** [review]

**Review by josephined21 — CHANGES_REQUESTED** [review]
for line 146 in placeholder.tsx, can you delete the comment?

**Review by mpark63 — APPROVED** [review]

**Pull request merged**